src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -15,7 +15,257 @@
     end
 
     it 'will protect forms without authenticity token' do
-      expect { post :save_person, project: "home:tom" }.to raise_error ActionController::InvalidAuthenticityToken
+      expect { post :save_person, project: 'home:tom' }.to raise_error ActionController::InvalidAuthenticityToken
     end
   end
+
+  describe 'GET #index' do
+    context 'showing all projects' do
+      before do
+        create(:confirmed_user, login: 'moi')
```

Comment:
"Showing all projects" means that home project are also included... so, I need a home project to make the difference between this context and the other